scipy/signal/_filter_design.py

```diff
@@ -5376,16 +5376,17 @@ def gammatone(freq, ftype, order=None, numtaps=None, fs=None):
     >>> from scipy import signal
     >>> import matplotlib.pyplot as plt
 
-    >>> b, a = signal.gammatone(440, 'iir', fs=16000)
+    >>> fc, fs = 440, 16000
```

Comment:
Should these two variables move to line 5367 and the 16 sampling example should also use these variables?

Reply:
I thought about it, but to me it looked like the idea was to keep the examples "separate". One can also remove the redundant imports on lines 5376 and 5377 and so on if that is preferred.

Just let me know what is preferred.